src/bg/user-script-registry.js

```diff
@@ -155,6 +155,24 @@ function onListUserScripts(message, sender, sendResponse) {
 }
 window.onListUserScripts = onListUserScripts;
 
+function onUserScriptGetByName(message, sender, sendResponse) {
+  if(!message.scriptNamespace) {
+    console.warn('UserScriptFind handler got no namespace')
+  } else if (!message.scriptName) {
+    console.warn('UserScriptFind handler got no name')
+  } else {
+    let userScriptIterator = UserScriptRegistry.scriptsToRunAt(null, true);
+    for (let userScript of userScriptIterator) {
+      if(userScript.namespace === message.scriptNamespace && userScript.name === message.scriptName) {
+        sendResponse(userScript.details);
+        return;
+      }
+    }
+    sendResponse(false);
```

Comment:
What response gets sent in the case of one of the "warn" blocks above?

Reply:
I have no idea. I just did this analogous to `onUserScriptGet`.